gunicorn_config.py

```diff
@@ -16,7 +17,7 @@
 # Guincorn sets the server type on our app. We don't want to show it in the header in the response.
 gunicorn.SERVER = "Undisclosed"
 
-on_aws = os.environ.get("NOTIFY_ENVIRONMENT", "") in [
```

Comment:
I don't know why we fell back on an empty string instead of the default `None` value. It seems unnecessary considering the following condition with the AWS env check.